armi/cases/inputModifiers/inputModifiers.py

```diff
@@ -53,10 +53,10 @@ def __call__(self, cs, blueprints, geom):
         else:
             coreBp = blueprints.gridDesigns["core"]
             coreBp.expandToFull()
-
-
-#             if self.eqFuelPathBp is not None:
-#                 self.eqFuelPathBp.expandToFull()
+            if "coreEqPath" in blueprints.gridDesigns:
+                # also expand core fuel management paths if they exist
+                eqPath = blueprints.gridDesigns["coreEqPath"]
+                eqPath.expandToFull()
```

Comment:
I dont know if this is generally safe to do... Maybe its okay as long as the center assembly never participates in shuffling.